src/gmusic-theme.js

```diff
@@ -101,6 +105,16 @@ window.GMusicTheme = class GMusicTheme {
           parent.appendChild((new DOMParser()).parseFromString(normalSVG, 'text/xml').firstChild);
         }
       }
+
+      // DEV: Google sometimes changes its logo byitself, we need to monitor this
+      this.observeLogo = new MutationObserver(() => {
```

Comment:
`observerLogo` sounds like a method. Maybe we should name it `this.logoObserver`?